src/widget/wtrackmenu.cpp

```diff
@@ -1762,6 +1779,17 @@ void WTrackMenu::slotRemoveFromDisk() {
         }
     }
 
+    // If the operation was initiated from a deck's track menu
+    // we'll first stop the deck and eject the track.
+    if (m_pTrack) {
+        ControlProxy* stopDeckProxy = new ControlProxy(
```

Comment:
You can use the static ControlObject function here:
ControlObject::set(ConfigKey(m_deckGroup, "stop"), 1.0);

Reply:
:+1: Thanks